### ISSUE TITLE
ID and aria attributes for accessibility

### ISSUE THREAD

**Issue by mquintSN** [jaywilz/react-bootstrap-range-slider#15]
This might be more of an enhancement, but an ID and aria attributes (`aria-valuemin`, `aria-valuemax`, `aria-valuenow`) should be added to the input element for accessibility.

Reference and example: 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_slider_role

I tried to pass an `id` prop but it didn't get rendered on the `<input>` html. I need an id on that to link a `<label>` to it.

**Comment by TomPlum** [comment]
+1.

We could do with additional props being forwarded to the underlying `<input />` component.

I'm wanting to add a `data-testid` attribute to be able to select it with React Testing Library and fire change events at it, but it doesn't forward the props so it never gets rendered in the DOM.

**Comment by jaywilz** [comment]
In version 3.0.0 the ARIA attributes (`aria-valuemin`, `aria-valuemax`, `aria-valuenow`) are added to the `<input>` and additional props are now passed through to the underlying `<input>`.

The `inputProps` prop is deprecated.

**Comment by TomPlum** [comment]
React 17 and TS support, ARIA attributes and props forwarding in one fell swoop... great work @jaywilz Thanks 😃
